spec/features/course/course_user_profile_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Courses: CourseUser Profile' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:course_student) { create(:course_student, :approved, course: course) }
+    let!(:achievement) { create(:course_user_achievement, course_user: course_student).achievement }
+    let(:course_teaching_assistant) do
+      create(:course_teaching_assistant, :approved, course: course)
+    end
+
+    before do
+      login_as(user, scope: :user)
+      visit course_course_user_path(course, course_user)
+    end
+
+    context 'As a Course Teaching Assistant' do
+      let(:user) { course_teaching_assistant.user }
+
+      context "When viewing a student's profile" do
```

Comment:
all it and context and scenario blocks do not start with a capital because they are all concatenated.